pkg/intfmapper/device.go

```diff
@@ -141,7 +141,7 @@ func (d *device) collect() error {
 		return nil
 	})
 	if err != nil {
-		return errors.Wrap(err, "BulkWalk failed")
```

Comment:
Why not return here? We would overwrite information we collected previously already with emptiness.